lib/fastlane/plugin/firebase_test_lab/options.rb

```diff
@@ -75,7 +75,13 @@ def self.available_options
                                        verify_block: proc do |value|
                                          v = File.expand_path(value.to_s)
                                          UI.user_error!("Key file not found at path '#{v}'") unless File.exist?(v)
-                                       end)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :ios_xc_test_args,
+                                       description: "extra ios_xc_test_args useful for xcode version ie { xcodeVersion: '10.1' }" \
```

Comment:
Suggested description:
```
Xcode version for running the tests.
```

This description is seen by the users and should be self explanatory.

Reply:
its not just for Xcode version for running tests though is it?